File: CON BDD/EUREKABANK_SOAP_JAVA_GR06/05. CLIESCRITORIO/eurekabank_soap_java_cliesc_gr06/src/main/java/ec/edu/monster/util/Moneda.java
package ec.edu.monster.util;

/** Catálogo de monedas (tabla `moneda`): muestra el nombre, no el código. */
public final class Moneda {

    private Moneda() { }

    public static String nombre(String codigo) {
        if (codigo == null) return "-";
        switch (codigo.trim()) {
            case "01": return "Soles";
            case "02": return "Dólares";
            default:   return codigo;
        }
    }
}

File: CON BDD/EUREKABANK_SOAP_DOTNET_GR06/06. CLIWEB/Util/Moneda.cs
namespace CLIWEB.Util
{
    public static class Moneda
    {
        public static string Nombre(string codigo)
        {
            return codigo switch
            {
                "01" => "Soles",
                "02" => "Dolares",
                _ => codigo
            };
        }
    }
}
